dev/io.openliberty.springboot.fat30_fat/fat/src/com/ibm/ws/springboot/support/fat/CommonWebFluxTests.java

```diff
@@ -84,12 +89,17 @@ public void testBlockingIO() throws IOException, InterruptedException {
 
     private static byte[] getResponse(HttpURLConnection con, int size) throws IOException, InterruptedException {
         ByteArrayOutputStream response = new ByteArrayOutputStream();
-        InputStream in = con.getInputStream();
-        byte[] buf = new byte[size];
-        int cnt;
-        while ((cnt = in.read(buf)) > 0) {
-            response.write(buf, 0, cnt);
-            Thread.sleep(sleep);
+        try {
+            InputStream in = con.getInputStream();
+            byte[] buf = new byte[size];
+            int cnt;
+            while ((cnt = in.read(buf)) > 0) {
+                response.write(buf, 0, cnt);
+                Thread.sleep(sleep);
+            }
+        } finally {
+            response.flush();
+            response.close();
```

Comment:
This is a local `ByteArrayOutputStream`.  I don't think we need a flush/close here.  But it doesn't hurt I guess.